test/parsers/parse-core.spec.ts

```diff
@@ -24,7 +24,10 @@ describe('when parsing .NET core', () => {
 
         const manifestFile = 'obj/project.assets.json';
         const expectedTree = JSON.parse(
-          fs.readFileSync(path.resolve(projectPath, 'expected.json'), 'utf-8'),
+          fs.readFileSync(
```

Comment:
Fixtures were mixing `expected` with `expected_depgraph`, so my consistency trigger kicked in.